server/drivers/hd44780-gpiod.c

```diff
@@ -35,19 +35,41 @@ void gpiod_HD44780_backlight(PrivateData *p, unsigned char state);
 void gpiod_HD44780_reset(PrivateData *p);
 void gpiod_HD44780_close(PrivateData *p);
 
+typedef struct {
+	uint offset;
+	struct gpiod_line_request *req;
+	struct gpiod_line_settings *settings;
+	struct gpiod_line_config *line_cfg;
+	struct gpiod_request_config *req_cfg;
+} gpio_pin;
+
 typedef struct {
 	struct gpiod_chip *chip;
-	struct gpiod_line *en;
-	struct gpiod_line *rs;
-	struct gpiod_line *d7;
-	struct gpiod_line *d6;
-	struct gpiod_line *d5;
-	struct gpiod_line *d4;
-	struct gpiod_line *en2;
-	struct gpiod_line *bl;
-	struct gpiod_line *rw;
+	gpio_pin *en;
+	gpio_pin *rs;
+	gpio_pin *d7;
+	gpio_pin *d6;
+	gpio_pin *d5;
+	gpio_pin *d4;
+	gpio_pin *en2;
+	gpio_pin *bl;
+	gpio_pin *rw;
 } gpio_pins;
 
+/**
+ * Set GPIO pin state.
+ *
+ * \param pin     Pointer to gpio_pin structure.
+ * \param value   The new PIN state to set.
+ */
+static int
+gpiod_line_set_value(gpio_pin *pin, int value)
```

Comment:
I'll rather prefer to choose option to rename that function, otherwise we would end up with calls like:
`gpiod_line_request_set_value(pin->req, pin->offset, (state == BACKLIGHT_ON) ? GPIOD_LINE_VALUE_ACTIVE : GPIOD_LINE_VALUE_INACTIVE);`